packages/hackathon/src/scenes/HomeScene/components/AgendaRow.tsx

```diff
@@ -53,7 +57,7 @@ export const AgendaRow: FC<AgendaCardProps> = ({ item, hacker, color }) => {
         </Span>
       </TableDataCell>
       <TableDataCell>
-        <ExtMarkdown source={item.description} />
+        <ExtMarkdown source={item.description} linkClickHandler={onClick} />
```

Comment:
Your onClick handler is exactly what's done in `ExtMarkdown` so let's create a TODO to clean this code back up after Hackathon is over.

Reply:
Yeah its an env adaptor problem. Something to fix later. 
```
Uncaught Error: Environment adaptor not initialized.
    at getEnvAdaptor (adaptorUtils.js:28:1)
    at openBrowserWindow (ExtMarkdown.js:13:1)
    at handleClick (Markdown.js:36:1)
```

Reply:
Sounds good.